components/warn_metric_ack_modal/warn_metric_ack_modal.tsx

```diff
@@ -141,51 +170,149 @@ export default class WarnMetricAckModal extends React.PureComponent<Props, State
+                                        url={learnMoreLink}
+                                        messageId={t('warn_metric_ack_modal.learn_more.link')}
+                                        defaultMessage={'Learn More'}
+                                    />
+                                ),
+                            }}
+                        />
+                    </div>
+                );
+            } else if (this.props.warnMetricStatus.id === WarnMetricTypes.SYSTEM_WARN_METRIC_NUMBER_OF_POSTS_500K) {
+                descriptionText = (
+                    <FormattedMessage
+                        id='warn_metric_ack_modal.number_of_posts.description'
+                        defaultMessage='TODO'
+                        values={{
+                            limit: this.props.warnMetricStatus.limit,
+                        }}
+                    />
+                );
+                subText = (
+                    <div
+                        style={{display: 'flex', opacity: '0.56', flexWrap: 'wrap'}}
+                        className='help__format-text'
+                    >
+                        <FormattedMessage
+                            id='warn_metric_ack_modal.number_of_posts.subtext'
+                            defaultMessage='TODO'
+                            values={{
+                                link: (
+                                    <ErrorLink
+                                        url={learnMoreLink}
+                                        messageId={t('warn_metric_ack_modal.learn_more.link')}
+                                        defaultMessage={'Learn More'}
+                                    />
+                                ),
+                            }}
+                        />
+                    </div>
+                );
+            }
+        }
+
+        let footer;
+        let error;
+
+        if (isE0Edition) {
+            error = this.renderStartTrialError();
+            footer = (<Modal.Footer>
+                <button
+                    className='btn btn-primary save-button'
+                    data-dismiss='modal'
+                    disabled={this.state.gettingTrial}
+                    autoFocus={true}
+                    onClick={this.onRequestLicenseAndAckWarnMetricClick}
+                >
+                    <LoadingWrapper
+                        loading={this.state.gettingTrial}
+                        text={Utils.localizeMessage('admin.warn_metric.getting_trial', 'Getting trial')}
+                    >
+                        <FormattedMessage
+                            id='warn_metric_ack_modal.start_trial'
+                            defaultMessage='Start trial'
+                        />
+                    </LoadingWrapper>
+                </button>
+            </Modal.Footer>);
+        } else {
+            error = this.renderContactUsError();
+            footer = (<Modal.Footer>
```

Comment:
Can we move this to a new line? Just for the purpose of making the JSX tags align properly.
```suggestion
            footer = (
                       <Modal.Footer>
```